index.d.ts

```diff
@@ -60,6 +60,7 @@ declare namespace contextMenu {
 
 	interface Actions {
 		readonly separator: () => MenuItem;
+		readonly services: () => MenuItem;
```

Comment:
Should be ordered after `inspect`, like in the menu. Same with the other places in the code.

Reply:
Done – I hadn't realised that was the intended ordering here.

Bearing that in mind, I seem to have put the entries for `lookUpSelection` in the wrong place too; shall I submit a PR to fix that?

Reply:
> shall I submit a PR to fix that?

👍 And also the labels thing.